crates/wallet/src/types.rs

```diff
@@ -72,7 +72,7 @@ pub struct WeightedUtxo {
     /// properly maintain the feerate when adding this input to a transaction during coin selection.
     ///
     /// [weight units]: https://en.bitcoin.it/wiki/Weight_units
-    pub satisfaction_weight: usize,
+    pub satisfaction_weight: usize, // FIXME: (@leonardo) I think it should expect the new bitcoin-units `Weight` type instead of usize.
```

Comment:
👍 agreed, now that there's a `Weight` type we should use it. If you think it's too big a change to add to this PR feel free to make an issue to fix it in a future PR.